xknx/conversion.py

```diff
@@ -0,0 +1,90 @@
+from datetime import time, date, datetime, timedelta
+from .colors import Colors
+from enum import Enum
+
+class xKNXCouldNotConvert(Exception):
+    def __init__(self):
+        pass
+
+    def __str__(self):
+        return "CouldNotConvert"
+
+class DPT():
+    def __init__(self, type = "", name = "", valuerange= (0,100), unit="", bitlength = 1):	
+	    self._type = type
+	    self._name = name
+	    self._valuerange = valuerange
+	    self._unit  = unit
+	    self._bitlength = bitlength
+    @property
+    def type(self):
+	    """Return the type"""
+	    return _type		
+    @property
+    def name(self):
+	    """Return the name"""
+	    return _name	
+    @property
+    def valuerange(self):
+	    """Return the valuerange"""
+	    return _valuerange	
+    @property
+    def unit(self):
+	    """Return the unit"""
+	    return _unit	
+    @property
+    def bitlength(self):
+	    """Return the unit"""
+	    return _bitlength			
+		
+class DPTTypes(Enum):
```

Comment:
wollte das ganze eigentlich nur als Template verwenden um die grenzen und werte zu bestimmen aber evtl können wir auch die Konvertierung direkt mit in die Klasse einbauen!?